app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
this method is not body.
so, you can write it once line
example: `def home; end`

Reply:
done